cobbler/items/item.py

```diff
@@ -1252,3 +1286,21 @@ def clean_cache(self, name: Optional[str] = None):
         :param name: The name of Item attribute or None.
         """
         self._clean_dict_cache(name)
```

Comment:
In the main branch here is the code:
https://github.com/cobbler/cobbler/blob/da8c2460f388603cc46e8002ece4d0fd0151c9c2/cobbler/items/item.py#L1336-L1337

but I suggest changing to:
```python
if self._inmemory and self.api.settings().cache_enabled:
    self._clean_dict_cache(name)
```
and remove the lines:
https://github.com/cobbler/cobbler/blob/da8c2460f388603cc46e8002ece4d0fd0151c9c2/cobbler/items/item.py#L1311-L1313